js/test/components/operatorSignIn/operatorSelection.test.tsx

```diff
@@ -110,4 +123,71 @@ describe("OperatorSelection", () => {
       method: "nfc",
     });
   });
+
+  test("executes onBadgeLookupError on unsuccessful badge tap result lookup", () => {
+    const onOK = jest.fn();
+    const onBadgeLookupError = jest.fn();
+
+    const { rerender } = render(
+      <OperatorSelection
+        onOK={onOK}
+        onBadgeLookupError={onBadgeLookupError}
+        onNfcScanError={jest.fn()}
+        nfcSupported={true}
+        employees={EMPLOYEES}
+      />,
+    );
+
+    jest.mocked(useNfc).mockReturnValueOnce({
```

Comment:
Can this be `jest.Mock` instead of `jest.mocked`?

Reply:
`jest.Mock<T>` is a type rather than a function so it's not something you can just swap in, and `useNfc` doesn't already have the right type, though I may be missing something you're saying.

Reply:
oh, I mean `jest.mock`, not `jest.Mock` 😅

Reply:
Oh! Uh, maybe, but its syntax is more verbose because you have to specify a module and everything.